src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 0 50%;
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 2px;
```

Comment:
do you really need margin on the left?
and don't forget about rightmost starts, they don't need `margin` 😉

<img width="239" alt="image" src="https://user-images.githubusercontent.com/55802630/172811996-8105fddb-fde3-4374-b72f-25d703e3c922.png">